Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.csproj

```diff
@@ -2,7 +2,27 @@
   <PropertyGroup>
     <TargetFramework>uap10.0.16299</TargetFramework>
     <Title>Windows Community Toolkit Controls DataGrid</Title>
-    <Description>This library provides a XAML DataGrid control. It is part of the Windows Community Toolkit.</Description>
+    <Description>
+      This library provides a XAML DataGrid control. It is part of the Windows Community Toolkit.
+
+      -CollectionViews : Allows handling CollectionViews List on a DataGrid.
+      -Datagrid :  Provides the neccesary files to provide a flexible way to display diferents types of data collections in rows and columns and to handle the related events. Managaes the distributions in cells, columns and rows of the displayed content.
+      -Datagrid.Automation : Contains the code to handle the AutomationPeers behaviour.
+      -Themes : Defines the ResourceDictionaries source path for the themes appliable to the control.
+      -Utilities : 
```

Comment:
These are all internal, so don't list them here.

Double-check for anything else called out?